apps/services/user-notification/infra/user-notification.ts

```diff
@@ -196,3 +198,31 @@ export const userNotificationCleanUpWorkerSetup = (): ServiceBuilder<
       staging: { schedule: '@midnight' },
       prod: { schedule: '@midnight' },
     })
+
+export const userNotificationBirthdayWorkerSetup = (services: {
+  userProfileApi: ServiceBuilder<'service-portal-api'>
+}): ServiceBuilder<typeof serviceBirthdayWorkerName> =>
+  service(serviceBirthdayWorkerName)
+    .image(imageName)
+    .namespace(serviceName)
+    .serviceAccount(serviceBirthdayWorkerName)
+    .codeOwner(CodeOwners.Juni)
+    .db({ name: 'user-notification' })
+    .command('node')
+    .args('--no-experimental-fetch', 'main.js', '--job=birthday')
+    .redis()
+    .env({ ...getEnv(services) })
+    .secrets({
+      FIREBASE_CREDENTIALS: `/k8s/${serviceName}/firestore-credentials`,
+      CONTENTFUL_ACCESS_TOKEN: `/k8s/${serviceName}/CONTENTFUL_ACCESS_TOKEN`,
+      IDENTITY_SERVER_CLIENT_ID: `/k8s/${serviceName}/USER_NOTIFICATION_CLIENT_ID`,
+      IDENTITY_SERVER_CLIENT_SECRET: `/k8s/${serviceName}/USER_NOTIFICATION_CLIENT_SECRET`,
+      NATIONAL_REGISTRY_B2C_CLIENT_SECRET:
+        '/k8s/api/NATIONAL_REGISTRY_B2C_CLIENT_SECRET',
+    })
+    .xroad(Base, Client, NationalRegistryB2C, RskCompanyInfo)
+    .extraAttributes({
+      dev: { schedule: '@midnight' },
+      staging: { schedule: '@midnight' },
+      prod: { schedule: '@midnight' },
```

Comment:
Maybe use a cronjob that is for 12'o clock every day?
What if we hit a boundary on the client machine such that we essentially skip a day or send notifications twice for some group?